README.md

```diff
@@ -109,11 +110,13 @@ from redact import RedactInstance, ServiceType, OutputType
 redact = RedactInstance.create(service=ServiceType.blur, out_type=OutputType.images, redact_url='https://api.brighter.ai/', api_key="VALID_API_KEY")
 ```
 
-In case you use a different api version than the default, you have to import the corresponding classes from the version module instead from `redact`:
+#### Different API version
+In case you use a different api version than the default, you have to import the corresponding classes from the version module instead from `redact`, e.g.:
 
 ```python
-from redact.v4 import RedactInstance, ServiceType, OutputType
+from redact.v3 import RedactInstance, ServiceType, OutputType, JobLabels
 ```
+> IMPORTANT! It is recommended to switch to API version v4, v3 will be removed in the future.   
```

Comment:
#### Switching API Versions in the client

When your application requires a specific API version that differs from the default provided by the SDK, you must import the relevant classes from that version's specific module. For instance, to use version 3 of the API, your import statement would look like this:

```python
# Importing classes from API version 3
from redact.v3 import RedactInstance, ServiceType, OutputType, JobLabels
```

> 🚨 **IMPORTANT**: API version v3 is currently deprecated and is scheduled for removal in the future. We encourage you to transition to API version v4 promptly to take advantage of the latest features, enhancements, and critical security updates.